### ISSUE TITLE
IS A MALWARE! 

### ISSUE THREAD

**Issue by hester95denis** [FDX100/GOD-KILLER#9]
Analyzed with Ghidra (reverse engineering tool)

**Comment by FDX100** [comment]
it's packaged with pyinstaller that's why it count as malware you can test it

**Comment by sayasss** [comment]
omg

**Comment by megalopsatlanticus** [comment]
Yeah bro...Ghidra will flag anything with a pyinstaller package as malware (as will most basic AV software deployed on server systems). Examine the source on your own...quit using tools. Or...ya know, do a simple SO search for related issues on this topic. I'm not exactly active on GitHub, but your 1 contribution doesn't bode well. Chill out...and do the background on your own.

**Comment by aminzikryy** [comment]
r u serius??

**Comment by CaraTortu** [comment]
It's not malware. I created a pull request with the original code by rever engineering this. You can see it's actually quite simple and not malware. If you upload it to virustotal or things it will tell you it is because normally all pyinstaller executables are flagged. You can run the code directly with the files in my pull request https://github.com/CaraTortu/GOD-KILLER
